### ISSUE TITLE
Bug `genai.list_models()`: TypeError: Model.__init__() got an unexpected keyword argument 'max_temperature'

### ISSUE THREAD

**Issue by dockhardman** [google-gemini/generative-ai-python#426]
## Environment

```
google-ai-generativelanguage-0.6.6
google-generativeai-0.7.1
```

## Reproduce Error

```python
# test_script.py
import google.generativeai as genai

genai.configure()
list(genai.list_models())
```

## Raise Error

```
Traceback (most recent call last):
  File "/Users/me/test_script.py", line 4, in <module>
    list(genai.list_models())
  File "/Users/me/miniconda3/envs/gls/lib/python3.10/site-packages/google/generativeai/models.py", line 206, in list_models
    yield model_types.Model(**model)
TypeError: Model.__init__() got an unexpected keyword argument 'max_temperature'
```

## Workaround

Constrained version `google-generativeai!=0.7.1`.

**Comment by pseudotensor** [comment]
Same problem, unsure why just started since python packages didn't change, seems issue with google endpoint itself.

**Comment by singhniraj08** [comment]
@dockhardman, Thank you for reporting this issue. `genai.list_models()` is broken in `0.7.1` and I was able to replicate it. Meanwhile we are fixing the issue, you can using generative-ai-python `0.7.0` release.

**Comment by chenin-wang** [comment]
Same problem

**Comment by maomihz** [comment]
+1

**Comment by maomihz** [comment]
Workaround, downgrade until they fix the issue in 0.7.1

pip install google-generativeai==0.7.0

**Comment by MarkDaoust** [comment]
Shoot! thanks for the report everyone. I'll try and get this fixed today.

**Comment by MarkDaoust** [comment]
A workaround to use 0.7.1 in the meantime is to install `google-ai-generativelanguage==0.6.5`.

**Comment by MarkDaoust** [comment]
This is fixed in the 0.7.2 release.
